backend/curriculum_tracking/models.py

```diff
@@ -1383,3 +1385,31 @@ def add_collaborator(self, user, add_as_project_reviewer):
                 if self.recruit_project:
                     self.recruit_project.reviewer_users.add(user)
         self.save()
+
+    def finding_latest_reviewer_users(self):
```

Comment:
If I ask you to tell me what this function does in English then the answer would be something like: 

"it gets the users that reviewed since the last review request" so a good name would be something like `get_users_that_reviewed_since_last_review_request`. It's more explicit and it's plain English.